server/models/verifiedUser.ts

```diff
@@ -0,0 +1,16 @@
+export class
+VerifiedUser{
```

Comment:
consider: `VerifiedUser` can extend the base `User` class.

Reply:
Updated `verifiedUser` to be an extension to the base class `User`